packages/noports_core/test/sshnp/util/sshrvd_channel/sshrvd_channel_test.dart

```diff
@@ -17,7 +17,7 @@ void main() {
     late SshrvGeneratorStub<String> sshrvGeneratorStub;
     late MockAtClient mockAtClient;
     late StreamController<AtNotification> notificationStreamController;
-    late FunctionStub notifyStub;
+    late NotifyStub notifyStub;
     late SubscribeStub subscribeStub;
```

Comment:
Wasn't verifying the arguments passed into notify, so I created a NotifyStub class which allows me to validate those arguments